README.md

```diff
@@ -1,205 +1,93 @@
-
-#### Run on Digital Ocean
-
-[![DigitalOcean Marketplace](assets/do-btn-blue.svg)](https://marketplace.digitalocean.com/apps/meilisearch?action=deploy&refcode=7c67bd97e101)
-
-#### Deploy on Platform.sh
-
-<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/meilisearch/.platform.template.yaml&utm_content=meilisearch&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
-    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
-</a>
+<!-- Go to [this demo]() to try Meilisearch! -->
 
-#### APT (Debian & Ubuntu)
-
-```bash
-echo "deb [trusted=yes] https://apt.fury.io/meilisearch/ /" > /etc/apt/sources.list.d/fury.list
-apt update && apt install meilisearch-http
-meilisearch
-```
-
-#### Download the binary (Linux & Mac OS)
-
-```bash
-curl -L https://install.meilisearch.com | sh
-./meilisearch
-```
-
-#### Compile and run it from sources
-
-If you have the latest stable Rust toolchain installed on your local system, clone the repository and change it to your working directory.
-
-```bash
-git clone https://github.com/meilisearch/meilisearch.git
-cd meilisearch
-cargo run --release
-```
-
-### Create an Index and Upload Some Documents
-
-Let's create an index! If you need a sample dataset, use [this movie database](https://www.notion.so/meilisearch/A-movies-dataset-to-test-Meili-1cbf7c9cfa4247249c40edfa22d7ca87#b5ae399b81834705ba5420ac70358a65). You can also find it in the `datasets/` directory.
-
-```bash
-curl -L https://docs.meilisearch.com/movies.json -o movies.json
-```
-
-Now, you're ready to index some data.
-
-```bash
-curl -i -X POST 'http://127.0.0.1:7700/indexes/movies/documents' \
-  --header 'content-type: application/json' \
-  --data-binary @movies.json
-```
-
-### Search for Documents
-
-#### In command line
-
-The search engine is now aware of your documents and can serve those via an HTTP server.
-
-The [`jq` command-line tool](https://stedolan.github.io/jq/) can greatly help you read the server responses.
+## ✨ Features
 
-```bash
-curl 'http://127.0.0.1:7700/indexes/movies/search?q=botman+robin&limit=2' | jq
-```
+- **Search-as-you-type experience** — Provides relevant search results in less than 50 milliseconds.
+- **Typo tolerant** — Understands misspellings and adapts to return relevant results.
+- **Facetting and filtering** — Enhance your user's search experience with custom filters and build a faceted search interface in a few lines of code.
+- **Sort by** — Sort results based on price, date, or pretty much anything else your users need.
+- **Synonyms** — Meilisearch supports synonyms customization, extending the relevant content your users find.
+- **Geosearch** — Filter, sort documents around real-world geographic data.
+- **Language support** — Meilisearch supports all languages and detects them automatically.
```

Comment:
Not sure about my sentence, but, this would be more honest.
```suggestion
- **Language support** — Meilisearch detects and supports all Latin-based languages and some more like Chinese or Japanese.
```